### DISCUSSION TITLE
Soliciting PR review

### DISCUSSION THREAD

**Discussion by ardavanhashemzadeh** [concourse/concourse#8896] [General]
Hello world,
Please forgive my ignorance as I'm not a regular contributor to opensource. I'm wondering who's attention I have to get to review/merge the following PR: https://github.com/concourse/resource-types/pull/87

The PR is to add to documentation a proposed 'concourse-webhook-resource', the resource is a very simple means of triggering a concourse pipeline via API call whilst throttling the frequency at which this happens. I tried to implement said functionality using only the existing git resource but was not successful (ended up creating an infinite loop type scenario from the commits to the tracking log creating new commits triggering new jobs).

**Comment by taylorsilva** [comment]
Done!
